### ISSUE TITLE
Damaged MyBook 2TB

### ISSUE THREAD

**Issue by MLUhle** [themaddoctor/linux-mybook-tools#59]
Hello,  My drive, tipped to the side and became corrupted.  I am able to download encrypted files to another drive, by using recovery software like EaseUS Data Recovery Wizard and photorec_win.  Is your software able to decrypt individual files?

**Comment by themaddoctor** [comment]
I don't believe that you could copy encrypted files, since the filesystem is also encrypted.

**Comment by MLUhle** [comment]
Hi, thanks for repliyng. 

 

First I tried cgsecurity.org ‘s  testdisk but it would not find any partitions.  Their other software Photorec found and I copied many files, some typical video files with sizes  > 3GB but the suffix doesn’t match, because of the encryption right?  Also the sizes don’t match my directory list I had  done before the crash, maybe because of the anterior reason,

 

From this list I have,  I see many videos that I already copied but, there are other things I would like to get back.  Another software EaseUS finds many files but I would have to buy a license in order to recover past 2GB.  I am willing to go this route if I can decrypt anything first.

 

Or should I just forget about it?

 

Thanks again

 

Mauro

**Comment by themaddoctor** [comment]
Sorry for my late reply, but I did not receive a notification that you had replied.

MyBooks that are encrypted use FULL-DISK encryption. This means that the file-system headers are also encrypted.
Furthermore, file-recovery software looks for file headers, but since they are encrypted, the software is finding
false positives.

If you want to recover your data, you have to decrypt the entire disk or set up a decryption filter on your operating system.
This project is simply an instruction manual for setting up such a filter on a linux computer. For full-disk decryption, see the
ReallyMine project at https://github.com/andlabs/reallymine That project can run on Mac and Windows as well.
